src/lang/Messages.properties

```diff
@@ -1429,7 +1429,7 @@ manReq.checkBox.followRedirect = Follow redirect
 manReq.checkBox.useSession     = Use current tracking session
 manReq.dialog.title            = Manual Request Editor
 manReq.pullDown.method         = Method
-manReq.resend.popup            = Resend
+manReq.resend.popup            = Manual Request Editor
```

Comment:
OK, just searched the zaproxy repo, worth adding a TODO comment mentioning that should be deleted once WebSockets is no longer using it.